scripts/zones/FeiYin/Zone.lua

```diff
@@ -101,7 +104,7 @@ function onEventFinish(player,csid,option)
         player:addKeyItem(LETTER_FROM_DALZAKK);
         player:messageSpecial(KEYITEM_OBTAINED,LETTER_FROM_DALZAKK);
     elseif (csid == 0x0017) then
-        player:setVar("SANDO92",3);
+        player:setVar("MissionStatus",3);

```

Comment:
No, this variable is used for the player nation Mission only, it gets resettled every time the lest mission ended. Also you can only accept one Nation mission at the time.